app/views/jobapps_mailer/saved_applications_notification.text.erb

```diff
@@ -0,0 +1,10 @@
+Hello,
+
+The following applications have been reactivated:
+<% @info.each_key do |position| %>
```

Comment:
Here too, if you need the value (which you do), better to iterate with `each_pair`.

It's surprising to me that Rubocop doesn't have a cop for this. [Perhaps we can write one of our own](https://github.com/umts/custom-cops).